vmm/src/vm.rs

```diff
@@ -1525,6 +1554,15 @@ impl<'a> Vm<'a> {
         Ok(())
     }
 
+    fn handler(signals: Signals, console_input_clone: Arc<vm_virtio::ConsoleInput>) {
```

Comment:
Please give this function a slightly more descriptive name.

Reply:
Done.